site/content/blog/lowrisc-6th-birthday-openuk-award/index.md

```diff
@@ -0,0 +1,21 @@
++++
+date = "2020-10-22T12:00:00+00:00"
+title = "A birthday present for lowRISC: We won an OpenUK Award!"
+featured_image = "lowrisc-openuk-award.jpg"
+
+[[resources]]
+src = "lowrisc-openuk-award.jpg"
+title = "lowRISC's 2020 OpenUK Award"
++++
+
+{{< img class="float-right rounded m-2" width=300 src="lowrisc-openuk-award.jpg" title="lowRISC's 2020 OpenUK Award" >}}
+
+On October 20th lowRISC CIC won in the Open Hardware category at the [2020 OpenUK Awards](https://openuk.uk/awards/), describing lowRISC as “the jewel in the crown of the UK’s open silicon companies”. The OpenUK awards promote “UK Leadership in Open Technology”, and are given out by OpenUK, a UK-based not-for-profit company which supports open source collaboration and open technologies within the United Kingdom.
+
+On receiving the award lowRISC CTO, Alex Bradbury, said “We're incredibly grateful to have been recognised for our achievements and contributions to date. Six years into our journey, it's a fantastic time to reflect on and thank the many people helping to make our vision for open source silicon a reality - our community of contributors, project partners, and the growing lowRISC team.”
```

Comment:
Nit: commas are wrong here. Should be "On receiving the award, lowRISC CTO Alex Bradbury said... " or "On receiving the award, Alex Bradbury, the lowRISC CTO, said... " (but that's more stilted). Or maybe "The lowRISC CTO, Alex Bradbury, welcomed the award, saying...".